set.mm

```diff
@@ -109155,7 +109155,7 @@ prove that every set is contained in a weak universe in ZF (see
        of Formalized Mathematics, Vol 1, No 3, May-August 1990.  A Tarski class
        is a set whose existence is ensured by Tarski's axiom A (see ~ ax-groth
        and the equivalent axioms).  Axiom A was first presented in Tarski's
-       article _&Uuml;ber unerreichbare Kardinalzahlen_.  Tarski introduced the
+       article _Ueber unerreichbare Kardinalzahlen_.  Tarski introduced the
```

Comment:
What'Äs the problem here with "_&Uuml;ber"?

Reply:
metamath-exe does not recognize `_` as starting italics unless the next character is alphabetic

Reply:
Before:

![image](https://github.com/metamath/set.mm/assets/16878/6f3cafc1-510e-443a-9842-27d21ddfad21)

After:

![image](https://github.com/metamath/set.mm/assets/16878/045de079-4971-4e2a-a373-a08dac7709bd)